cpp/src/round_robin/round_robin.cu

```diff
@@ -13,6 +13,20 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+#include <cudf/types.hpp>
+#include <cudf/table/table.hpp>
+#include <cudf/table/table_device_view.cuh>
+#include <cudf/table/row_operators.cuh>
+#include <cudf/utilities/type_dispatcher.hpp>
+#include <rmm/thrust_rmm_allocator.h>
```

Comment:
There's an `rmm` header that sneaked in here

Reply:
That was already here and it's technically necessary for the `device_vector`.

Reply:
Ok, it looked like you were reorganizing the headers to put cudf stuff at the top, so I wasn't sure if you meant to separate out the rmm headers